engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -3004,6 +3015,15 @@ void paladin_t::assess_damage( school_e school, result_amount_type dtype, action
     trigger_holy_shield( s );
   }
 
+  // On a block event, trigger T28 4p if equipped
+  // todo: Woli -  Set bonus check
+  if ( ( s->block_result == BLOCK_RESULT_BLOCKED )  && ( sets->has_set_bonus( PALADIN_PROTECTION, T28, B4 ) ) &&
+       ( rng().roll( tier_sets.glorious_purpose_4pc->effectN( 1 ).percent() ) ) ) 
+  //)
+  {
+    trigger_t28_4p_pp( s );
+  }
+
```

Comment:
```suggestion
  // On a block event, trigger T28 4p if equipped
  // todo: Woli -  Set bonus check
  if ( ( s->block_result == BLOCK_RESULT_BLOCKED )  && sets->has_set_bonus( PALADIN_PROTECTION, T28, B4 ) 
        && rng().roll( tier_sets.glorious_purpose_4pc->effectN( 1 ).percent() ) ) 
  {
    trigger_t28_4p_pp( s );
  }

```
Trim some parenthesis, add a new line, remove some commented clutter